### DISCUSSION TITLE
Early-checked include paths

### DISCUSSION THREAD

**Discussion by shiide** [Codeception/Codeception#6176] [Ideas]
I feel that a test run shouldn't start with any invalid include paths. I propose checking for validity of include paths early on configuration load.

I'm not aware of any issue this fail-early method would cause, please correct me if I'm wrong. Also, I'm not aware of any benefit of allowing tests to run before an include failure.

Failure should also include information on exactly which include is the culprit.

**Comment by DavertMik** [comment]
Could you be more specific about include paths? What do you mean? (better to point to line of code you actually refering to)

**Comment by shiide** [comment]
I mean checking existence & readability of include paths given in codeception.yml configs: a47db8191a0481d45b6dbfb22170aad30b8926e9

**Comment by Naktibalda** [comment]
@DavertMik How do you like this proposal?
